<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LootTableEntry</class>
 <widget class="QTabWidget" name="LootTableEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>254</height>
   </rect>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::South</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <property name="usesScrollButtons">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="general">
   <attribute name="title">
    <string>General</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="5">
     <widget class="MultiPageWidget" name="multiPageWidget">
      <widget class="QWidget" name="page_2">
       <property name="windowTitle">
        <string>Empty</string>
       </property>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="windowTitle">
        <string>Item</string>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Item:</string>
          </property>
          <property name="buddy">
           <cstring>itemSlot</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="InventorySlot" name="itemSlot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <property name="windowTitle">
        <string>Loot table</string>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="buddy">
           <cstring>tableNameEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="tableNameEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <property name="windowTitle">
        <string>Tag</string>
       </property>
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="buddy">
           <cstring>tagNameEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="tagNameEdit"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="tagExpandCheck">
          <property name="text">
           <string>Expand</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <property name="windowTitle">
        <string>Dynamic</string>
       </property>
       <layout class="QFormLayout" name="formLayout_7">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="buddy">
           <cstring>dynamicNameEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="dynamicNameEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_7">
       <property name="windowTitle">
        <string>Group</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Select mode</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QRadioButton" name="selectSeqRadio">
             <property name="text">
              <string>Until the first entry fails</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="selectAllRadio">
             <property name="text">
              <string>All entries</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="selectAltRadio">
             <property name="text">
              <string>Only the first successful entry</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="entryGroupLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add and remove entries in the &lt;a href=&quot;/entries&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Entries&lt;/span&gt;&lt;/a&gt; tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="indent">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Weight:</string>
      </property>
      <property name="buddy">
       <cstring>weightSpin</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="weightSpin">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="qualityLabel">
      <property name="toolTip">
       <string>Bonus weight from luck.</string>
      </property>
      <property name="text">
       <string>Quality:</string>
      </property>
      <property name="buddy">
       <cstring>qualitySpin</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QSpinBox" name="qualitySpin">
      <property name="minimum">
       <number>-999</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="entries">
   <attribute name="title">
    <string>Entries (%1)</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="DataWidgetInterface" name="entriesInterface">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="functions">
   <attribute name="title">
    <string>Functions (%1)</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="DataWidgetInterface" name="functionsInterface">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="condtions">
   <attribute name="title">
    <string>Conditions (%1)</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="DataWidgetInterface" name="conditionsInterface">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiPageWidget</class>
   <extends>QWidget</extends>
   <header>multipagewidget.h</header>
   <addpagemethod>addPage</addpagemethod>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DataWidgetInterface</class>
   <extends>QFrame</extends>
   <header location="global">datawidgetinterface.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InventorySlot</class>
   <extends>QFrame</extends>
   <header>inventoryslot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableNameEdit</tabstop>
  <tabstop>tagNameEdit</tabstop>
  <tabstop>tagExpandCheck</tabstop>
  <tabstop>dynamicNameEdit</tabstop>
  <tabstop>selectAllRadio</tabstop>
  <tabstop>selectAltRadio</tabstop>
  <tabstop>selectSeqRadio</tabstop>
  <tabstop>weightSpin</tabstop>
  <tabstop>qualitySpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
